client/signup/steps/user-type/form.jsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class UserTypeForm extends Component {
+	static propTypes = {
+		userType: PropTypes.string,
+		submitForm: PropTypes.func.isRequired,
+
+		// from localize() HoC
+		translate: PropTypes.func.isRequired,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			userType: props.userType,
```

Comment:
Is this the only reason why we have the `userType` prop? If that's the case, let's completely remove it, because we don't seem to be using it at all, so we can just define a `null` or `''` here.